static/src/js/util/accessMethods/__tests__/buildAccessMethods.test.js

```diff
@@ -19,38 +32,82 @@ describe('buildAccessMethods', () => {
     }
     const isOpenSearch = false
 
-    const methods = buildAccessMethods(collectionMetadata, isOpenSearch)
+    buildAccessMethods(collectionMetadata, isOpenSearch)
 
-    expect(methods).toEqual({
-      download: {
-        isValid: true,
-        type: 'download'
-      }
-    })
+    expect(buildDownloadMock).toBeCalledTimes(1)
+
+    expect(buildDownloadMock).toBeCalledWith({ items: [{ online_access_flag: true }] }, false)
   })
 
-  test('returns a download access method for open search', () => {
-    const collectionMetadata = {}
-    const isOpenSearch = true
+  describe('when ordering is disabled', () => {
+    test('calls buildEcho access method and no echo-order access method is returned', () => {
+      const buildEsiEchoMock = jest.spyOn(buildEsiEcho, 'buildEsiEcho')
+      buildEsiEchoMock.mockImplementationOnce(() => jest.fn())
 
-    const methods = buildAccessMethods(collectionMetadata, isOpenSearch)
+      jest.spyOn(getApplicationConfig, 'getApplicationConfig').mockImplementation(() => ({
+        disableOrdering: 'true'
+      }))
 
-    expect(methods).toEqual({
-      download: {
-        isValid: true,
-        type: 'download'
+      const collectionMetadata = {
+        services: {
+          items: [{
+            type: 'ECHO ORDERS',
+            url: {
+              urlValue: 'https://example.com'
+            },
+            maxItemsPerOrder: 2000,
+            orderOptions: {
+              items: [{
+                conceptId: 'OO10000-EDSC',
+                name: 'mock form',
+                form: 'mock form'
+              }]
+            }
+          }]
+        }
       }
+      const isOpenSearch = false
+
+      buildAccessMethods(collectionMetadata, isOpenSearch)
+
+      expect(buildEsiEchoMock).toBeCalledTimes(1)
+
+      expect(buildEsiEchoMock).toHaveBeenCalledWith({
+        maxItemsPerOrder: 2000,
+        orderOptions: {
+          items: [
+            {
+              conceptId: 'OO10000-EDSC',
+              form: 'mock form',
+              name: 'mock form'
+            }
+          ]
+        },
+        type: 'ECHO ORDERS',
+        url: {
+          urlValue: 'https://example.com'
+        }
+      }, {
+        associatedVariables: {},
+        echoIndex: 0,
+        esiIndex: 0,
+        harmonyIndex: 0
+      })
     })
   })
 
-  test('returns an esi access method', () => {
+  test('calls buildEsiEcho access method with type ECHO ORDERS', () => {
```

Comment:
These tests should be organized like this, "When <some input parameter>, then <expected output>" For this example:
```
describe('when the method type is ECHO ORDERS', () => {
  test('calls buildEsiEcho', () => {
    ...
  })
})
```